modules/channel-web/src/views/lite/components/messages/MessageGroup.tsx

```diff
@@ -66,17 +71,32 @@ class MessageGroup extends React.Component<Props> {
           {this.props.showUserName && <div className={'bpw-message-username'}>{this.props.userName}</div>}
           <div className={'bpw-message-group'}>
             {this.props.messages.map((data, i) => {
+              const isLastMsg = i == this.props.messages.length - 1
+              const payload = this.convertPayloadFromOldFormat(data)
+
+              let collectFeedback = payload.collectFeedback
+              if (payload.wrapped) {
+                collectFeedback = payload.wrapped.collectFeedback
+              }
+
               return (
                 <Message
                   key={`msg-${i}`}
                   isHighlighted={
                     this.props.highlightedMessages && this.props.highlightedMessages.includes(data.incomingEventId)
                   }
+                  extra={
+                    this.props.isBot &&
+                    collectFeedback &&
+                    isLastMsg && (
```

Comment:
Since extra is only being used for the inline feedback, I suggest renaming it `inlineFeedback`

Maybe also move the conditional logic in a variable? `const inlineFeedback = (payload.collectFeedback || payload.wrapped?.collectFeedback) && this.props.isBot && isLastMsg `